### ISSUE TITLE
Weird behavior with transpose-frame.el

### ISSUE THREAD

**Issue by IceAsteroid** [localauthor/org-side-tree#12]
Demo: org-side-tree with transpose-frame

https://github.com/localauthor/org-side-tree/assets/90975914/537c69c5-bf66-4a4a-85c1-1bb51555b80d

After that, I also tried to disable perfect-margin which is a package to keep content of a buffer composed at center.

Demo: transpose-frame with other windows

https://github.com/localauthor/org-side-tree/assets/90975914/7e1af6b6-de0c-42c2-b55c-a6087371888f

**Comment by localauthor** [comment]
I’m not familiar with transpose-frame, but it looks like an unmaintained package. Maybe it doesn’t work well with side windows? There’s no issue history for that package, so without digging into the source code I wouldn’t be able to say what the problem is.

**Comment by IceAsteroid** [comment]
You're right. I remember it wrong and thought treemacs as a sidebar is able to work fine with ```transpose-frame.el```, and coincidentally treemacs & org-sidebar hadn't been working after an update, so I thought it might be a problem specific to org-side-tree. but it's not. transpose-frame basically misbehaves with different types of sidebars.

I'm gonna close this issue
